### PR TITLE
Form expired messages

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes add messages for expired forms on the static pages and on the form introduction pages.

## Testing done
unit tests updated

## Screenshots
Intro page
image
image

Static pages
image
image

## Acceptance criteria
- [x] Display a message to users if their form has expired.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile360/util/helpers.jsx [hunk 1]
```diff
@@ -31,6 +31,23 @@ export const formTitles = Object.keys(formBenefits).reduce((titles, key) => {
   return titles;
 }, {});
 
+export const formDescriptions = Object.keys(formBenefits).reduce(
+  (titles, key) => {
+    let formNumber;
+    if (key === '40-10007' || key === 'VIC') {
+      formNumber = '';
+    } else if (key === '1010ez') {
+      formNumber = ' (10-10EZ)';
```
Comment (rianfowler) [inline src/applications/personalization/profile360/util/helpers.jsx]: does this need a space?

**Review by rianfowler — APPROVED** [review]

File: src/applications/personalization/profile360/util/helpers.jsx (hunk 1 above)
Comment (cehsu) [reply src/applications/personalization/profile360/util/helpers.jsx]: Good idea, @rianfowler, we could move the space to be after application, that seems clearer.

**Comment by peggygannon** [comment]
Hi, @cehsu I have some additional edits to the messages 

- Can we please add "Your" before the titles? I think the "your" got cut off in my edits when I cut/pasted. sorry about that!
Your form has expired
Your form is in progress

*Intro page message* 
- The expired message in the screenshot, doesn't exactly match what I posted. So the second sentence should read: If you want to apply for health care benefits, please start a new application


- Also I had suggested we update the button to "Start a New Application" Is that not possible?

- The form in progress message should say (it looks like Your got dropped in the second sentence)
Your {benefit} application (VA Form {number}) is in progress.
Your application was last saved on {date} at {time a.m. or p.m.}

- Also I had suggested we update the button to "Start a New Application" Is this not possible?

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu Hi, Claire. Thanks for the updated screenshots. Just a few more edits: 

Expired view for static pages: 
There is an extra "application" in the message. Can you delete the 2nd one, so the message says what's below. 
also please insert a space between "for" and "health"
Your form has expired
Your saved health care benefits application (10-10EZ) has expired. If you want to apply for health care benefits, please start a new application. 
button: Start a New Application


Introduction page messages: 
Can we remove the last line "Please complete the 10-10EZ form to apply for health care benefits"
It's repetitive and I don't think we need that line in the messages.

**Comment by cehsu** [comment]
Thanks @peggygannonm kindly refer to the updated screenshots:
## Static page expired view
screen shot 2018-10-16 at 10 45 41 am

## Introduction page saved view
screen shot 2018-10-16 at 10 46 10 am

**Review by peggygannon — APPROVED** [review]

**Pull request merged**